aiida/cmdline/verdilib.py

```diff
@@ -947,18 +947,6 @@ def _check_db_name(self, dbname, method=None, **kwargs):
         return dbname, create
 
 
-class Runserver(VerdiCommand):
```

Comment:
Can you remove this also from the docs? to a search with 'git grep -i runserver', but I think there are at least a few parts - the documentation of verdi, and then probably also this can be removed: http://aiida-core.readthedocs.io/en/latest/setup/aiida_multiuser.html#deploying-aiida-using-apache